Earthfile

```diff
@@ -37,6 +37,7 @@ proto:
   ' gen/proto/go/translate/v1/translate.pb.gw.go
 
   RUN rm gen/proto/go/translate/v1/translate.pb.gw.go.bak
+  #TODO Save to more apropriate dir e.g. pkg/pb/translate/v1 or pkg/gen/translate/v1
```

Comment:
In the future want to refractor the structure a bit for more clarity:
1. Change `api/translate/translate/v1/translate.proto` to `proto/translate/v1/translate.proto`
2. Change `pkg/server/translate/v1...` to `pkg/pb/translate/v1...`

Too many changes for this PR